content/rs/references/rladmin.md

```diff
@@ -1,30 +1,786 @@
+        [ persistent_path <path> ]
+        [ ccs_persistent_path <path> ]
+        [ rack_id <node-rack-id> ]
+        [ override_rack_id ]
+        [ replace_node <node-uid> ]
+        [ flash_enabled ]
+        [ flash_path <path> ]
+        [ addr <ip-address> ]
+        [ external_addr <ip-addresses> ]
+        [ override_repair ]
+        [ accept_servers <enable | disable> ]
+        [ cmn_http_port <port> ]
+```
+
+| Optional Parameters | Description |
+| - | - |
+| ephemeral_path | Path to ephemeral storage location (defaults to /var/opt/redislabs)|
+| persistent_path | Path to persistent storage location (defaults to /var/opt/redislabs/persist) |
+| ccs_persistent_path | Path to ccs snapshot location (defaults to /var/opt/redislabs/persist) |
+| rack_id | Rack ID of the rack |
+| override_rack_id | Manually overrides the existing default rack ID |
+| replace_node | Replaces specified node with new node |
+| flash_enabled | Enables flash storage |
+| flash_path | Path to flash storage location (defaults to /var/opt/redislabs/flash) |
+| addr | Internal IP addresses of the node |
+| external_addr | External IP addresses of the node |
+| override_repair | Enables joining a cluster with a dead node |
+| accept_servers | Allows allocation of resources on the new node |
+| cmn_http_port | Joins a cluster that has a non-default cnm_http_port |
+
+#### `cluster recover`
+
+`rladmin cluster recover` recovers a cluster from a backup file. Configuration backup file's default location is `/var/opt/redislabs/persist/ccs/ccs-redis.rdb`.
+
+```text
+rladmin cluster recover 
+        filename <recovery-file-name> 
+        [ ephemeral_path <path> ] 
+        [ persistent_path <path> ]
+        [ ccs_persistent_path <path> ]
+        [ ccs_persistent_path <path> ]
+        [ rack_id <ID> ] 
+        [ override_rack_id ] 
+        [ node_uid <number> ] 
+        [ flash_enabled ] 
+        [ flash_path <path> ] 
+        [ addr <ip-address> ] 
+        [ external_addr <ip-addresses> ]
+```
+
+| Optional Parameters | Description |
+| - | - |
+| ephemeral_path | Path to ephemeral storage location (defaults to /var/opt/redislabs) |
+| persistent_path | Path to persistent storage location (defaults to /var/opt/redislabs/persist) |
+| ccs_persistent_path | Path to location of ccs snapshots (default is same as persistent_path) |
+| rack_id | Rack ID of the rack |
+| override_rack_id | Manually overrides the existing default rack ID |
+| flash_enabled | Enables flash storage |
+| flash_path | Path to flash storage location (defaults to /var/opt/redislabs/flash) |
+| addr | Internal IP addresses of the node |
+| external_addr | External IP addresses of the node |
+
+#### `cluster services`
+
+`rladmin cluster services` enables or disables selected cluster services.
+
+```text
+rladmin cluster services [ cm_server | crdb_coordinator | crdb_worker | mdns_server | pdns_server | saslauthd | stats_archiver <enabled | disabled> ]
+```
+
+### `failover`
```

Comment:
@AlonMagrafta  : Mark to myself, I reviewed this PR to this point, not including the extended part of cluster create+recover+join